app/base/models.py

```diff
@@ -87,10 +91,29 @@ class Property(db.Model):
     image_folder = db.Column(db.Text, nullable=True)
     photos = db.Column(db.Text)
     user_id = db.Column(db.ForeignKey("User.id"), nullable=False)
+    owner = db.relationship('User')
+
+    def to_son(self):
+        """
+        This function returns the a json representation of the object. This is useful if we want to include and
+        return data related to the instance from other database tables
+        """
+        return {
+            "id": self.id,
+            "name": self.name,
+            "desc": self.desc,
+            "price": self.price,
+            "date": self.date,
+            "location": self.location,
+            "image_folder": self.image_folder,
+            "photos": self.photos,
+            "user_id": self.user_id,
+            "owner": user_schema.dump(self.prop_owner)
+        }
 
     @staticmethod
     def get_all_properties():
-        return [properties_schema.dump(prop) for prop in Property.query.all()]
+        return [Property.to_son(prop) for prop in Property.query.all()]
```

Comment:
this  typo  seems it was not  resolved

Reply:
My bad. I resolve the conversation before commit and pushing the changes you requested.
Am still getting an an error when I push my commits.